_data/jobs.yml

```diff
@@ -1,4 +1,9 @@
-- expires: 2022-01-07
+- expires: 2022-01-30
+  location: RAND: Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington D.C.
+  name: Research Software Engineer (Entry Level)
+  posted: 2021-12-07
+  url: https://rand.wd5.myworkdayjobs.com/en-US/External_Career_Site/job/Santa-Monica-CA-Greater-Los-Angeles-Area/Research-Software-Engineer--Entry-Level-_R772
+ - expires: 2022-01-07
```

Comment:
I can't tell if a space was added here to change the indentation (it looks weird on my screen) but if it was, probably want to remove that or it will mess up the yaml parsing.

Reply:
ugh, thanks - the space is an auto indent issue - thought I got it, but I didn't delete enough.  Gets me every time :(

Reply:
That one gets me too :sob: